src/main/app/src/hooks/formValidation.js

```diff
@@ -0,0 +1,27 @@
+/** This react hook contains functions used to validate event information entered on the
+    schedule activity page. */
+
+const isValidEmail = email => {
+  const re = /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
+  return re.test(email);
+}
+
+const validate = (title, startTime, endTime, updateTitleError, updateDateError) => {
+  let errors = false;
+
+  // validate title
+  if (title === "") {
+    updateTitleError(true);
+    errors = true;
+  }
+
+  // validate times
+  if (startTime.getTime() > endTime.getTime()) {
```

Comment:
Not necessary, but would you like to combine these two if statements with an OR operator, since their bodies do the same thing and manipulate the same states? We can save some lines of duplicate code. :)

Reply:
It's not the same because in the first conditional statement I'm updating updateTitleError while the second one is for updateDateError. They are two different checks.